flybywiresim/a32nx
# fix(atsu): handle multiple alternates

Fixes #8034
Fixes #8006 

## Summary of Changes

Fixes the loading of Simbrief flight plans with multiple alternates by choosing the first one. 


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): Chaoz#3065

## Testing instructions


1. Generate Simbrief flight plan with multiple alternate airports
2. Load flight plan from MCDU
3. Ensure no errors appear and that alternate is correctly chosen as the first one defined on Simbrief


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
